### PR TITLE
Update DatePicker docs

### PR DESCRIPTION
**Pull request by ztplz**
First of all, thank you for your contribution! 😄

New feature please send pull request to feature branch, and rest to master branch.
Pull request will be merged after one of collaborators approve.
Please makes sure that these form are filled before submitting your pull request, thank you!

[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / document update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Branch merge
- [ ] Other (about what?)

### What's the background?

> 1. Describe the source of requirement.
> 2. Resolve what problem.
https://github.com/ant-design/ant-design/issues/14761
https://github.com/ant-design/ant-design/issues/13433
> 3. Related issue link.

### API Realization (Optional if not new feature)

> 1. Basic thought of solution and other optional proposal.
> 2. List final API realization and usage sample.
> 3. GIF or snapshot should be provided if includes UI/interactive modification.

### What's the effect? (Optional if not new feature)

> 1. Does this PR affect user? Which part will be affected?
> 2. What will say in changelog?
> 3. Does this PR contains potential break change or other risk?

### Changelog description (Optional if not new feature)

> 1. English description
> 2. Chinese description (optional)

### Self Check before Merge

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

### Additional Plan? (Optional if not new feature)

> If this PR related with other PR or following info. You can type here.

---

<details>
<summary>
<b>IssueHunt Summary</b>
</summary>

### Referenced issues

This pull request has been submitted to:
- [#13433: RangePicker missing from value will cause app crashed if clicking on the component](https://oss.issuehunt.io/repos/34526884/issues/13433)
---
</details>

### CODE REVIEW

**Comment by ztplz** [comment]
Land after https://github.com/react-component/calendar/pull/520

File: components/date-picker/index.zh-CN.md [hunk 1]
```diff
@@ -119,7 +119,7 @@ moment.locale('zh-cn');
 
 | 参数 | 说明 | 类型 | 默认值 |
 | --- | --- | --- | --- |
-| defaultValue | 默认日期 | [moment](http://momentjs.com/)\[] | 无 |
+| defaultValue | 默认日期，如果开始时间或结束时间为 null，日期范围将是一个开区间| [moment](http://momentjs.com/)\[] | 无 |
```
Comment (afc163) [inline components/date-picker/index.zh-CN.md]: 支持 `undefined` 么？

Comment (ztplz) [reply components/date-picker/index.zh-CN.md]: 不支持，我按null判断的，undefined需要支持吗

Comment (afc163) [reply components/date-picker/index.zh-CN.md]: undefined 更合理一点，语义上是开始/结束日期**未指定**。

File: components/date-picker/index.en-US.md [hunk 2]
```diff
@@ -116,7 +116,7 @@ The following APIs are shared by DatePicker, MonthPicker, RangePicker, WeekPicke
 
 | Property | Description | Type | Default |
 | -------- | ----------- | ---- | ------- |
-| defaultValue | to set default date, if start time or end time is null, the date range will be an open interval | \[[moment](http://momentjs.com/), [moment](http://momentjs.com/)] | - |
+| defaultValue | to set default date, if start time or end time is null or undefined, the date range will be an open interval | \[[moment](http://momentjs.com/), [moment](http://momentjs.com/)] | - |
```
Comment (afc163) [inline components/date-picker/index.en-US.md]: null undefined 加 `

**Comment by afc163** [comment]
ping~

**Comment by afc163** [comment]
diff 有点奇怪

**Comment by ztplz** [comment]
我这个merge不知道merge了什么东西

**Pull request merged**